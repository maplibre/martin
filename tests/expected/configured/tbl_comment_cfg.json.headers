content-encoding: br
content-type: application/json
etag: W/"136-lHzPvh-7gSnjgJnd-2ctzQ=="
transfer-encoding: chunked
vary: accept-encoding, Origin, Access-Control-Request-Method, Access-Control-Request-Headers
