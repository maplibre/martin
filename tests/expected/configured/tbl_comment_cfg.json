{
  "bounds": [
    -170.94985961914062,
    -84.20025634765625,
    167.7089385986328,
    74.23573303222656
  ],
  "description": "a description from comment on table",
  "name": "MixPoints",
  "tilejson": "3.0.0",
  "tiles": [
    "http://localhost:3111/MixPoints/{z}/{x}/{y}"
  ],
  "vector_layers": [
    {
      "fields": {
        "Gid": "int4",
        "TABLE": "text"
      },
      "id": "MixPoints"
    }
  ]
}
